### PR TITLE
updated server.py to use json file

### PR DESCRIPTION
**Pull request by Divi03**

### CODE REVIEW

File: backend/server.py [hunk 1]
```diff
@@ -8,8 +8,12 @@
 def hello():
     return "Hello World!"
 
+fl = open('backend/data.json','r')
+json_data = json.load(fl)
+
 @app.route("/records")
 def records():
+
```
Comment (Khalid-212) [inline backend/server.py]: remove the list and replace it with the json file

**Review by Khalid-212 — CHANGES_REQUESTED** [review]
remove the list and replace it with the json file

**Review by Divi03 — COMMENTED** [review]
removed list

**Review by Khalid-212 — APPROVED** [review]
good job

**Pull request merged**